msu/utilities/exceptions.nut

```diff
@@ -0,0 +1,9 @@
+::MSU.Exception <- {
```

Comment:
do we wanna add a way of adding Exceptions?

Reply:
No. If some modder wants a new exception, they can suggest and we can add it to MSU codebase. I don't want modders polluting the `::MSU.Exception` namespace, potentially even having conflicts. They can make their own exceptions in their own namespaces, or ask us to add something here.